src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js

```diff
@@ -63,6 +63,8 @@ const Body = BaseFormWithPolling.extend({
     let challengeRequest = deviceChallenge.challengeRequest !== undefined ? deviceChallenge.challengeRequest : '';
     let probeTimeoutMillis = deviceChallenge.probeTimeoutMillis !== undefined ?
       deviceChallenge.probeTimeoutMillis : 100;
+    let isEnhancedPollingEnabled = deviceChallenge.enhancedPollingEnabled !== undefined
+      && deviceChallenge.enhancedPollingEnabled === true;
```

Comment:
nit:
```suggestion
    const isEnhancedPollingEnabled = typeof deviceChallenge.enhancedPollingEnabled === 'boolean'
      && deviceChallenge.enhancedPollingEnabled;
```

It doesn't look like `isEnhancedPollingEnabled` is reassigned, but correct me if I am wrong, but if not, should use const instead of let.

Reply:
@glenfannin-okta we can have a future enhancement to put all possible values as const. Right now we will just keep it as let for consistency and also because we have tested it end to end. @SohamKolhatkar-okta please create a Jira for this purpose.

Reply:
See my comment below, if you use the exact code change I provided this line shouldn't be necessary. Additionally in general any `typeof` check and checks for `undefined` are unnecessary if we directly do a strict equals/not-equals check against a boolean.

Reply:
Sure, will create a JIRA tomorrow, when I have time.